hugegraph-server/hugegraph-dist/src/assembly/static/bin/common-topling.sh

```
@@ -0,0 +1,286 @@
+        fi
+    fi
+}
+
+function download_and_verify() {
+    local url=$1
+    local filepath=$2
+    local expected_md5=$3
+
+    if [[ -f $filepath ]]; then
+        echo "File $filepath exists. Verifying MD5 checksum..."
+        actual_md5=$(md5sum $filepath | awk '{ print $1 }')
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)